guides/source/getting_started.md

```diff
@@ -172,7 +172,7 @@ of the files and folders that Rails created by default:
 |app/|Contains the controllers, models, views, helpers, mailers, channels, jobs and assets for your application. You'll focus on this folder for the remainder of this guide.|
 |bin/|Contains the rails script that starts your app and can contain other scripts you use to setup, update, deploy or run your application.|
 |config/|Configure your application's routes, database, and more. This is covered in more detail in [Configuring Rails Applications](configuring.html).|
-|config.ru|Rack configuration for Rack based servers used to start the application.|
+|config.ru|Rack configuration for [Rack](https://rack.github.io/) based servers used to start the application.|
```

Comment:
`For more information about Rack, see the [Rack website](https://rack.github.io/).` like Bundler and Yarn.